cmd/curio/guidedsetup/guidedsetup.go

```diff
@@ -237,20 +246,66 @@ type MigrationData struct {
 	owner           address.Address
 	worker          address.Address
 	sender          address.Address
-	ssize           abi.SectorSize
+	ssize           string
 	confidence      uint64
 	init            bool
 }
 
 func complete(d *MigrationData) {
 	stepCompleted(d, d.T("Lotus-Miner to Curio Migration."))
-	d.say(plain, "Try the web interface with %s for further guided improvements.", code.Render("curio run --layers=gui"))
+}
+
+var EnvFiles = []string{"/etc/curio/config.env", "./curio/curio.env", "~/config/curio.env"}
+
+func afterRan(d *MigrationData) {
+	// Write curio.env file. TODO!!!!!!!!!!
+	// Inform users they need to copy this to /etc/curio.env or ~/.config/curio.env to run Curio.
+	places := append([]string{"/tmp/curio.env",
+		must.One(os.Getwd()) + "/curio.env",
+		must.One(os.UserHomeDir()) + "/curio.env"}, EnvFiles...)
+saveConfigFile:
+	_, where, err := (&promptui.Select{
+		Label:     d.T("Where should we save your database config file?"),
+		Items:     places,
+		Templates: d.selectTemplates,
+	}).Run()
+	if err != nil {
+		d.say(notice, "Aborting migration.", err.Error())
+		os.Exit(1)
+	}
+
+	args := []string{fmt.Sprintf("CURIO_DB=postgres://%s:%s@%s:%s/%s",
+		d.HarmonyCfg.Username,
+		d.HarmonyCfg.Password,
+		d.HarmonyCfg.Hosts[0],
+		d.HarmonyCfg.Port,
+		d.HarmonyCfg.Database)}
+
+	if e := os.Getenv("FULLNODE_API_INFO"); e == "" {
+		addr, header, err := cliutil.GetRawAPI(d.cctx, repo.FullNode, "v1")
+		if err != nil {
+			d.say(notice, "Add FULLNODE_API_INFO to curio.env: %s", err.Error())
+		} else {
+			args = append(args, fmt.Sprintf("FULLNODE_API_INFO=%s:%s", addr, header.Get("Authorization")[8:]))
+		}
+	} else {
+		args = append(args, fmt.Sprintf("FULLNODE_API_INFO=%s", e))
+	}
```

Comment:
We can remove this as I explained in slack. This doesn't do anything for us. Daemon is picked up from DB anyway.